packages/acceptance-tests/pkg-tests-specs/sources/commands/config.test.js

```diff
@@ -1,34 +1,39 @@
 const {
-  fs: {mkdirp, writeFile},
+  fs: {mkdirp, writeFile, createTemporaryFolder},
 } = require('pkg-tests-core');
 
 const RC_FILENAME = `.spec-yarnrc`;
 const SUBFOLDER = `subfolder`;
 const FAKE_REGISTRY_URL = `http://yarn.test.registry`;
 const FAKE_LOCAL_APP_DATA = `LOCAL_APP_DATA`;
 const FAKE_WORKSPACE_ROOT = `WORKSPACE_ROOT`;
+const FAKE_HOME = `HOME`;
 
 const environments = {
   [`folder without rcfile in ancestry`]: async () => {
     // Nothing to do
   },
-  [`folder with rcfile`]: async path => {
+  [`folder with rcfile`]: async ({path}) => {
     await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
   },
-  [`folder with rcfile without trailing newline`]: async path => {
+  [`folder with rcfile without trailing newline`]: async ({path}) => {
     await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test`);
   },
-  [`folder with rcfile and rc in parent`]: async path => {
+  [`folder with rcfile and rc in parent`]: async ({path}) => {
     await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
     await writeFile(`${path}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
   },
-  [`folder with rcfile and rc in ancestor parent`]: async path => {
+  [`folder with rcfile and rc in ancestor parent`]: async ({path}) => {
     await writeFile(`${path}/${SUBFOLDER}/${SUBFOLDER}/${RC_FILENAME}`, `init-scope berry-test\n`);
     await writeFile(`${path}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
   },
+  [`folder with rcfile and rc in home folder`]: async ({path, homePath}) => {
+    await writeFile(`${homePath}/${RC_FILENAME}`, `init-scope berry-test\n`);
+    await writeFile(`${path}/${RC_FILENAME}`, `init-scope berry-test\nlastUpdateCheck 1555784893958\n`);
```

Comment:
I think there's a problem in this test case (and possibly the one above as well): you seem to have no way to know whether the rcfile from the home folder got parsed or not (since it uses the same value as the one in the current directory it will be indistinguishable between "doesn't work but regular rc works" and "doesn't work"). I suggest to:

- use a different value for `init-scope`
- also add a different option (for example `defaultLanguageName python`).